src/utilities/focus.ts

```diff
@@ -0,0 +1,153 @@
+import { isElementInViewport } from '@polaris/utilities/is-element-in-viewport';
```

Comment:
Does this file include any `react` code inside? 
If not, should we import this directly from polaris instead of copy to?